core/embed/rust/src/protobuf/encode.rs

```diff
@@ -220,7 +219,7 @@ impl<'a> OutputStream for BufferStream<'a> {
                 *pos += len;
                 buf.copy_from_slice(val);
             })
-            .ok_or(Error::Missing)
+            .ok_or(error::end_of_buffer())
```

Comment:
`ok_or_else`

Reply:
fixed in b406bd0a0ff986392c2fccef336a7b9201734023